src/wmtk/operations/tri_mesh/VertexLaplacianSmooth.hpp

```diff
@@ -30,8 +30,6 @@ class VertexLaplacianSmooth : public VertexAttributesUpdateBase
 
     std::string name() const override;
 
-    static PrimitiveType primitive_type() { return PrimitiveType::Vertex; }
```

Comment:
Is there a reason why you removed those? Is it deprecated code?

Reply:
I just checked the tupleoperation class and it has a broken use of this function. We should make an independent issue to fix this